scripts/i18n/update-source.js

```diff
@@ -19,36 +23,42 @@ const dirsToCopy = [
 ]
 
 function cloneOrUpdateRepo(repoName, repoUrl) {
+  shell.cd(path.join(originRepo, `..`))
   if (shell.ls(repoName).code !== 0) {
-    logger.debug(`cloning ${repoName}`)
-    shell.exec(`git clone ${repoUrl}`)
+    logger.debug(`Cloning ${repoName}`)
+    shell.exec(`git clone --quiet --depth 1 ${repoUrl} > /dev/null`)
```

Comment:
Is it important to have it log not print anything? I kind of like it printing *something* since that's easier to debug if something goes wrong

Reply:
If `git clone` failures, it'd log the failure and include the full url (which would include the `GITHUB_API_TOKEN` and Circle logs are public) 

That's why we pipe this to `/dev/null` specifically